### PR TITLE
Run command options + added URL param to SAPI client instances

### PR DESCRIPTION
**Pull request by ujovlado**
(Replaces https://github.com/keboola/docker-bundle/pull/116)

Related to: https://github.com/keboola/syrup-queue/pull/41

### CODE REVIEW

File: Docker/Container.php [hunk 1]
```diff
@@ -79,31 +84,31 @@ public function getImage()
...
         $minLogPort,
-        $maxLogPort
+        $maxLogPort,
+        RunCommandOptions $runCommandOptions
     ) {
         $this->logger = $logger;
         $this->containerLogger = $containerLogger;
         $this->image = $image;
         $this->dataDir = $dataDirectory;
         $this->id = $containerId;
```
Comment (odinuv) [inline Docker/Container.php]: tak `protected $environmentVariables = [];` by melo jit do haje

Comment (odinuv) [inline Docker/Container.php]: a taky na r. 222 se to pouziva

Comment (ujovlado) [inline Docker/Container.php]: ou, to mi uslo ... to este fixnem

**Comment by ujovlado** [comment]
@odinuv myslim, ze toto je ready ... test mi znehli aj lokalne (az na nejake storage writer, co je asi tym, ze bezim v `eu-central-1`), tak by som to pustil na Jenkinsa ako branch a asi mergol.

**Comment by odinuv** [comment]
na jenkinsu uz ti to bezi, dopoledne jsem to nastavil :)

**Pull request merged**